Cisco/cisco-secure-web-appliance/tests/test_export_to_database.json

```diff
@@ -0,0 +1,34 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco Secure Web Appliance",
+        "dialect_uuid": "23b75d0c-2026-4d3e-b916-636c27ba4931"
+      }
+    },
+    "message": "Info: Completed writing export files to database (#counter_group: WEB_APPLICATION_TYPE_APPLICATION_NAME_DETAIL #interval 2023-02-10-11-40 #Serial number: 10F920FF9B46-WZP254808V0 #Time since data generated: 369\n"
+  },
+  "expected": {
+    "message": "Info: Completed writing export files to database (#counter_group: WEB_APPLICATION_TYPE_APPLICATION_NAME_DETAIL #interval 2023-02-10-11-40 #Serial number: 10F920FF9B46-WZP254808V0 #Time since data generated: 369\n",
```

Comment:
```suggestion
    "message": "Info: Completed writing export files to database (#counter_group: WEB_APPLICATION_TYPE_APPLICATION_NAME_DETAIL #interval 2023-02-10-11-40 #Serial number: 123456-789101112 #Time since data generated: 369\n",
```